waiting/src/main/resources/properties/monitoring.yml

```
@@ -46,8 +46,8 @@ spring:
   config.activate.on-profile: prod
 management:
   server:
-    port: 19043
-    address: 172.31.38.190
+#    port: 19043
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)